go/arrow/csv/writer.go

```diff
@@ -44,9 +45,10 @@ func NewWriter(w io.Writer, schema *arrow.Schema, opts ...Option) *Writer {
 	validate(schema)
 
 	ww := &Writer{
-		w:         csv.NewWriter(w),
-		schema:    schema,
-		nullValue: "NULL", // override by passing WithNullWriter() as an option
+		boolFormatter: strconv.FormatBool,
```

Comment:
maybe add a similar comment to the `nullValue` comment ie: `// override by passing WithBoolFormatter as an option`

Reply:
done in [b8bcc18](https://github.com/apache/arrow/pull/13774/commits/b8bcc1856b4a3589c608107a65c7fb267ec8df4d)